README.md

```diff
@@ -292,6 +292,16 @@ govuk-docker compose run mysql mysql -h mysql -u root --password=root -e "CREATE
 pv whitehall_production.dump.gz | gunzip | govuk-docker compose run mysql mysql -h mysql -u root --password=root whitehall_development
 ```
 
+*For Whitehall*
+
+In order to ensure that Whitehall runs smoothly, your Docker settings may need to be changed in order to handle Whitehall. By default, Docker defaults to using 2 CPU cores and 2GB of RAM, and this is simply not enough to render Whitehall in a Docker container.
+
+You should set it to use at least 4 CPU cores and 8GB of RAM - 6 CPUs and 8 GB of RAM seems to work ok. Open the Docker dropdown via the whale icon in the OSX menu bar and selecting the `Advanced` option.
```

Comment:
I'd suggest that these settings apply beyond just Whitehall and are pretty much a pre-requisite to running govuk-docker. 

Maybe rather than giving 2 numbers (min and ideal) we just suggest an ideal and it's at a user's risk if they go lower. I'd actually suggest 6 cores and 12GB of ram rather than 8GB.

Reply:
Done in https://github.com/alphagov/govuk-docker/pull/182/commits/c9a83d33effc7f0402cdeb636eb9f2a512f402b9